src/main/java/entities/Customer.java

```diff
@@ -1,11 +1,24 @@
 package entities;
 
 public class Customer {
+
     private int customerId;
     private boolean isAdmin;
     private String customerName;
-    private String LastName;
-    private String personId;
+    private String customerLastName;
```

Comment:
we can have just name or first and last name

Reply:
Ok, understand